src/main/java/ca/corefacility/bioinformatics/irida/service/impl/analysis/submission/AnalysisSubmissionServiceImpl.java

```diff
@@ -831,4 +831,12 @@ public Long getAnalysesRanInTimePeriod(Date createdDate) {
 		return analysisSubmissionRepository.countAnalysesRanGrouped(createdDate, statisticTimePeriod.getGroupByFormat());
 	}
 
+	/**
+	 * {@inheritDoc}
+	 */
+	@PreAuthorize("hasRole('ROLE_ADMIN') or hasRole('ROLE_USER')")
```

Comment:
Isn't ROLE_ADMIN just a super set of the ROLE_USER?  Do you need to authorize both?

Reply:
Good catch. Updated in 0042a0ad2e7a59b91ddeb0ff9efe5adaa46c6c25